fawltydeps/packages.py

```diff
@@ -194,32 +210,54 @@ def lookup_package(self, package_name: str) -> Optional[Package]:
         return self.packages.get(Package.normalize_name(package_name))
 
 
+class IdentityMapping(BasePackageResolver):
+    """An imperfect package resolver that assmues package name == import name.
+
+    This will resolve _any_ package name into a corresponding identical import
+    name (modulo normalization, see Package.normalize_name() for details).
+    """
+
+    def lookup_package(self, package_name: str) -> Optional[Package]:
+        """Convert a package name into a Package with the same import name."""
+        ret = Package.identity_mapping(package_name)
+        logger.info(
+            f"Could not find {package_name!r} in the current environment. "
+            f"Assuming it can be imported as {', '.join(sorted(ret.import_names))}"
+        )
+        return ret
+
+
 def resolve_dependencies(
     dep_names: Iterable[str], pyenv_path: Optional[Path] = None
 ) -> Dict[str, Package]:
     """Associate dependencies with corresponding Package objects.
 
-    Use LocalPackageLookup to find Package objects for each of the given
+    Use LocalPackageResolver to find Package objects for each of the given
     dependencies inside the virtualenv given by 'pyenv_path'. When 'pyenv_path'
     is None (the default), look for packages in the current Python environment
     (i.e. equivalent to sys.path).
 
-    For dependencies that cannot be found with LocalPackageLookup,
-    fabricate an identity mapping (a pseudo-package making available an import
-    of the same name as the package, modulo normalization).
+    For dependencies that cannot be found with LocalPackageResolver, fall back
+    an identity mapping (a pseudo-package making available an import of the same
+    name as the package, modulo normalization).
```

Comment:
I think with this refactor we do not need this sentence. Resolution order is defined in `resolvers` list.
```suggestion
```